recipes/transformer-smaller-training-vocab/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "transformer-smaller-training-vocab" %}
+{% set version = "0.2.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/transformer_smaller_training_vocab-{{ version }}.tar.gz
+  sha256: a87a2b3a0f345210f8d2a20dfb15d624f43d93e5cf35425faf99e0e6bdd5a709
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7,<4.0
+    - poetry-core
+    - pip
+  run:
+    - python >=3.7,<4.0
+    - datasets =>2.0.0,<3
+    - transformers =>4.1,<5
+    - pytorch =>1.8.0,<2
+    - sentencepiece >=0.1.97,<0.2
```

Comment:
Not sure about this because Grayskull failed me but the deps are here: https://github.com/helpmefindaname/transformer-smaller-training-vocab/blob/06a53c0f1c3cc6863fe4295d25c158c35184f7af/pyproject.toml#L11